src/Widgets/DeviceItem.vala

```diff
@@ -111,7 +111,8 @@ namespace Network.Widgets {
                         break;
                 }
 
-                subtitle = Utils.state_to_string (state);
+                subtitle = device is NM.DeviceWifi && state == NM.DeviceState.UNAVAILABLE ?
+                            _("Disabled") : Utils.state_to_string (state);
```

Comment:
Since this is already a long line, it's probably more clear to use if/else.

```suggestion
                if (device is NM.DeviceWifi && state == NM.DeviceState.UNAVAILABLE) {
                    subtitle = _("Disabled");
                } else {
                    subtitle =  Utils.state_to_string (state);
                }
```

Reply:
Looks better, done!